backend/tests/e2e/e2e_utils.py

```
@@ -0,0 +1,311 @@
+        Setup HTTP mocking for runbook sources.
+
+        Args:
+            respx_mock: The respx mock object
+            content: Content to return for runbook requests
+        """
+        respx_mock.get(url__regex=r".*(github\.com|runbooks\.example\.com).*").mock(
+            return_value=httpx.Response(200, text=content)
+        )
+
+    @staticmethod
+    def create_mcp_client_patches(mock_sessions: Dict[str, AsyncMock]) -> Tuple[callable, callable]:
```

Ruff (0.12.2)

Line too long (100 > 88)

(E501)